nixos/modules/programs/sway.nix

```diff
@@ -28,22 +28,29 @@ let
 in {
   options.programs.sway = {
     enable = mkEnableOption ''
-      the tiling Wayland compositor Sway. After adding yourself to the "sway"
-      group you can manually launch Sway by executing "sway" from a terminal.
-      If you call "sway" with any parameters the extraSessionCommands won't be
-      executed and Sway won't be launched with dbus-launch'';
+      Sway, the i3-compatible tiling Wayland compositor
```

Comment:
We should probably keep the following:

> You can manually launch Sway by executing "sway" from a terminal.

(TODO: systemd user units and support for login managers)

The behaviour of `extraSessionCommands` is more intuitive now and it is probably better not to document it (or just say that these commands should not take too long and should be safe to run multiple times (e.g. `extraSessionCommands` will also be executed when running `sway --version` on a virtual terminal).